### DISCUSSION TITLE
Automations / External Data Connector / Bindings have no JS

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7437] [Help] [answered]
Ver : 1.2.47 / Self / Docker

The Automations / External Data Connector / Bindings can only take HBS and not JS.
Is this a bug or how its supposed to be ?

**Comment by melohagan** [comment, marked as answer]
Thanks for that @poirazis 

It is a bug! Raised here: https://github.com/Budibase/budibase/issues/7473
